
Tweak the default config to get installbuilddir right.

Upstream-Status: should really make the "RedHat" layout DTRT again and
		use that layout instead

--- httpd-2.1.10/config.layout.layout
+++ httpd-2.1.10/config.layout
@@ -20,7 +20,7 @@
     mandir:        ${prefix}/man
     sysconfdir:    ${prefix}/conf
     datadir:       ${prefix}
-    installbuilddir: ${datadir}/build
+    installbuilddir: ${libdir}/apache2/build
     errordir:      ${datadir}/error
     iconsdir:      ${datadir}/icons
     htdocsdir:     ${datadir}/htdocs
